SLHCUpgradeSimulations/Geometry/test/phase2_digi_reco_pixelntuple_cfg.py

```diff
@@ -19,7 +19,7 @@
 process.load('Configuration.StandardSequences.DigiToRaw_cff')
 process.load('HLTrigger.Configuration.HLT_Fake2_cff')
 process.load('Configuration.StandardSequences.RawToDigi_cff')
-process.load('Configuration.StandardSequences.L1Reco_cff')
+#process.load('Configuration.StandardSequences.L1Reco_cff')
```

Comment:
this can be removed for good, right?

Reply:
Right. We already remove it from the main Phase-2 sequence.

Here is the issue we discussed,
https://github.com/cms-sw/cmssw/issues/33012

Reply:
so instead of commenting can you delete the line?

Reply:
Delete.